cli/cmd/generate.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// generateCmd represents the generate command
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Use for generating utilities",
```

Comment:
It'll be helpful to keep the description consistent.
The other sub-command added in this command is used for generating Rundeck resources; it may be safe to say that this is
> Use for generating resources